backend/CircleForms/Controllers/ApiController.cs

```diff
@@ -22,7 +22,7 @@ public ApiController(ILogger<ApiController> logger, IUserDatabaseService usersSe
         [HttpGet]
         public async Task<string> Get()
         {
-            return string.Join(',', await _usersService.Get());
+            return string.Join(',', (await _usersService.Get()).Select(x=> x.Id));
```

Comment:
I think it's better to execute `.ToList()` on every single database call to force evaluation despite `_userService.Get()` calls `.ToListAsync`.
If the api of userRepository will be changed to be lazy evaluated it WILL cause problems.